components/gitpod-db/src/typeorm/migration/1679909342076-AddOrganizationSettings.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddOrganizationSettings1679909342076 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            "CREATE TABLE IF NOT EXISTS d_b_team_settings (  teamId char(36) NOT NULL,  disabledWorkspaceSharing tinyint(4) NOT NULL DEFAULT '0', PRIMARY KEY (teamId)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;",
```

Comment:
Will also need the `_deleted` property, such that it can be set, and an async process can delete the data

Reply:
And possibly a CreatedAt and UpdatedAt properties